x-element.js

```diff
@@ -1040,10 +1040,10 @@ class TemplateEngine {
    * @returns {any}
    */
   static html(strings, ...values) {
-    const reference = TemplateEngine.#createReference();
+    const symbol = Symbol();
```

Comment:
Again, we could have say like `Symbol('html')` or `Symbol('result')` here — but I really don’t think that’s additive. If anything, it makes it seem like the string has meaning when it really doesn’t for our use-case.